packages/google_maps_flutter/google_maps_flutter/pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: google_maps_flutter
 description: A Flutter plugin for integrating Google Maps in iOS and Android applications.
 homepage: https://github.com/flutter/plugins/tree/master/packages/google_maps_flutter/google_maps_flutter
-version: 0.5.25+3
+version: 0.5.25+4
```

Comment:
0.5.26 as the PR is adding a new feature.
https://github.com/flutter/plugins/pull/2650